### PR TITLE
Support login with email (MVP)

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: src/resdk/resolwe.py [hunk 1]
```diff
@@ -502,7 +503,10 @@ def __init__(self, username=None, password=None, url=DEFAULT_URL):
         if not username and not password:
             return
 
-        payload = {"username": username, "password": password}
+        if is_email(username):
```
Comment (mstajdohar) [inline src/resdk/resolwe.py:506]: How about something like this?

```
key = "email" if is_email(username) else "username"
payload = {key: username, "password": password}
```

**Review by mstajdohar — APPROVED** [review]
Splendid, thank you

**Comment by mstajdohar** [comment]
Do we need to update the docs too?

File: src/resdk/resolwe.py (hunk 1 above)
Comment (JureZmrzlikar) [reply src/resdk/resolwe.py]: Good suggestion, thanks!

**Comment by JureZmrzlikar** [comment]
> Do we need to update the docs too?
I've updated the login prompt.

**Pull request merged**